app/code/community/Sendsmaily/Sync/etc/system.xml

```diff
@@ -132,6 +132,68 @@
               </depends>
             </sync_period>
 
+            <newsletter_form_section translate="label">
+              <label>Newsletter form</label>
+              <frontend_model>adminhtml/system_config_form_field_heading</frontend_model>
+              <sort_order>40</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>0</show_in_store>
+            </newsletter_form_section>
+
+            <active_newsletter_form translate="label comment">
+              <label>Enabled</label>
+              <comment><![CDATA[Collect newsletter subscribers from newsletter form. Learn more from our <a href=http://help.smaily.com/en/support/solutions/articles/16000092458-creating-automation-workflows#a4 target=_blank>tutorial how to create welcome emails.</a>]]></comment>
```

Comment:
Maybe a slightly better link placement would be `Learn more from our tutorial on <a href="..." />how to create welcome emails</a>.`.

Reply:
Changed in db5f1a6025fc752c7739437d38b3644d68299912